boards/seeed/xiao_esp32s3/doc/index.rst

```diff
@@ -1,12 +1,12 @@
 .. _xiao_esp32s3:
 
-XIAO ESP32S3
-############
+XIAO ESP32S3 (Sense)
```

Comment:
I would suggest listing both board names like this to avoid ambiguity:
```suggestion
XIAO ESP32S3/XIAO ESP32S3 Sense
```

Reply:
done